utils/polus-python-template/README.md

```diff
@@ -1,140 +1,99 @@
+
+The code generated provides out-of-box support for :
+    - customizing the plugin code.
+    - implementing tests.
+    - creating and running a container.
+    - managing versioning.
+    - updating documentation (README, CHANGELOG).
+    - maintaining a WIPP manifest (plugin.json).
+
+## Customizing the plugin code
+
+The plugin should be run as a package.
+The skeleton code can be run this way :
+From the plugin's top directory :
+`cd src`
+`python -m polus.package1.package2.awesome_function -i /tmp/inp -o /tmp/out`
+
+This should print some logs with the provided inputs and outputs and return.
+
+## Implementing tests
 
-** NOTE: ** Do not modify `project_slug`. This is automatically generated from
-the name of the plugin. If your plugin is called "Awesome Segmentation", then
-the plugin folder and docker container will have the name
-`polus-awesome-segmentation-plugin`.
+Plugin's developer should preferably uses `pytest`.
+A simple test has been added to the template as an example.
+Before submitting a PR to `polus-plugins`, other unit tests should be created and added to the `tests`
+directory.
 
-## What's New?
+To run tests :
+From the plugin's top directory, type `python -m pytest`.
+This should run a test successfully and return.
 
-## bump2version
+
+## Managing versioning
 
 Making sure that the file version is consistent across files in a plugin can be
 challenging, so the Python template now uses
 [bump2version](https://github.com/c4urself/bump2version)
 to help manage versioning. This automatically changes the `VERSION` and
 `plugin.json` files to the next version, preventing you from having to remember
-to change the version in all 3 places. The `bumpversion.cfg` can be modified to
+to change the version everywhere. The `bumpversion.cfg` can be modified to
 change the version in other files as well.
 
 To use this feature:
 1. `pip install bump2version`
 2. `bump2version --config-file bumpversion.cfg`
-3. If an error is thrown about uncommited work, use the `--alow-dirty` option
-
-## unittest
-
-A simple `unittest` has been added to the template that verifies the `VERSION`
-matches the `plugin.json` version information. Before submitting a PR to
-`polus-plugins`, other unit tests should be created and added to the `tests`
-directory. The test classes that are created should be added to the tuple
-`test_cases` in `tests/__init__.py`.
-
-## run-plugin.sh
-
-A new shell script (`run-plugin.sh`) has been added to help setup testing of
-docker containers locally. It comes configured with the proper `docker run`
-command already set up so that input variables just need to be defined.
-
-## Explanation of File Structure and Settings
-
-### General Structure
-
-In general, the structure of a plugin should have the following files as a minimum:
+3. If an error is thrown about uncommited work, use the `--allow-dirty` option
```

Comment:
This should be a secondary option. The user should first try to add and commit the changes to git.